.github/templates/macos_binary_build_workflow.yml.j2

```diff
@@ -17,7 +17,9 @@ name: !{{ build_environment }}
           # shellcheck disable=SC2129
           echo "BINARY_ENV_FILE=${RUNNER_TEMP}/env" >> "${GITHUB_ENV}"
           # shellcheck disable=SC2129
-          echo "PYTORCH_FINAL_PACKAGE_DIR=${RUNNER_TEMP}/artifacts" >> "${GITHUB_ENV}"
+          echo "PYTORCH_FINAL_PACKAGE_DIR=${RUNNER_TEMP}/artifacts/wheel" >> "${GITHUB_ENV}"
```

Comment:
Can you please describe why it's safe to change the path that artifacts get uploaded from, esp during the non-last loop?